guides/extensions/react-hooks/index.md

```diff
@@ -0,0 +1,22 @@
+# React Hooks for Aries Framework JavaScript
+
+The `react-hooks` package exposes useful React hooks that allow you to easily interact with AFJ.
+
+Everything exported from React Hooks:
+
+```ts
+import AgentProvider, {
```

Comment:
Not sure if I understand the flow here. The index contains code on usage, but you need to go to setup for installation?

Reply:
Right. I'll fix that